edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Core/Constants.cs

```diff
@@ -105,6 +105,8 @@ public static class Constants
 
         public const string StorageMaxTotalWalSize = "RocksDB_MaxTotalWalSize";
 
+        public const string StorageLogLevel = "Storage_LogLevel";
```

Comment:
Would StoreAndForward_LogLevel naming be more clear?

Reply:
1. I think the structure (or tuble) `StoreAndForward` should really be broken into 2 tuples.
2. Since the LogLevel is only use for the RocksDb (and not for RoutingModule or the other module), I think the Storage_LogLevel is reasonable. 

Now that I think about it, I think I will break the `StoreAndForward` into `StoreConfig` and `???Config`

Reply:
I think `Storage_LogLevel` is justified since the config does not use for the `Forwarding` part. The parameter is only used for `CommonModule` but not for `RoutingModule` and `MqttModule`